src/OpenTelemetry.Instrumentation.Http/Implementation/HttpWebRequestActivitySource.netfx.cs

```diff
@@ -166,13 +166,21 @@ private static string GetErrorType(Exception exception)
             {
                 WebExceptionStatus.NameResolutionFailure => "name_resolution_failure",
                 WebExceptionStatus.ConnectFailure => "connect_failure",
+                WebExceptionStatus.ReceiveFailure => "receive_failure",
```

Comment:
Did you refer to any doc to get these new types?

Reply:
If yes, could you add the details in the CHANGELOG?

Reply:
I did not. As per the spec instrumentation can define the error.type they can set. We would need to document the possible values in Readme though as suggested by spec.

https://github.com/open-telemetry/semantic-conventions/blob/main/docs/http/http-spans.md#:~:text=The%20error.type%20value%20SHOULD%20be%20predictable%20and%20SHOULD%20have%20low%20cardinality.%20Instrumentations%20SHOULD%20document%20the%20list%20of%20errors%20they%20report.